src/features/slackapi/slackapi-utils.ts

```diff
@@ -169,11 +169,24 @@ Have a great week!
 
       const message = constructDailyMessage(userData, numberOfToday);
 
+      const listOfIds = process.env.STAGING_IDS.split(",");
+
+
       if (!isAway && !firstDayBack) {
-        messageResults.push({
-          message: message,
-          response: await sendMessage(slackId, message.message)
-        });
+        if (listOfIds.length === 0) {
+          messageResults.push({
+            message: message,
+              response: await sendMessage(slackId, message.message)
+              });
+              }
+        else {
+          for (stagingid of listOfIds){
```

Comment:
```suggestion
          for (const stagingid of listOfIds){
```

Reply:
In a for of loop the variable should be declared (as above), again make sure everything still works as intended after changes